actions/command.test.js

```diff
@@ -72,10 +72,12 @@ const initialState = {
             bindings: [{
                 location: '/command',
                 bindings: [{
+                    location: '/command/appid',
                     app_id: 'appid',
                     label: 'appid',
                     bindings: [
                         {
+                            location: '/command/appid/custom',
```

Comment:
@larkox, asked the same in a diff. context (your blog post) - but do we allow the  user define smth. like '/command/appid/custom'? (like are we protecting against that)?

Reply:
We are protecting ourselves of locations lower than the base (i.e. if you call your command location `/command/something`, it will be sent back as `/command/command/something`. Since the locations are something used by the app developer, and we only use the base when we use it, there is no "danger" on whatever the app developer does there.